scripts/west_commands/runners/jlink.py

```diff
@@ -167,16 +174,20 @@ def do_run(self, command, **kwargs):
             client_cmd = (self.gdb_cmd +
                           self.tui_arg +
                           [self.elf_name] +
-                          ['-ex', 'target remote :{}'.format(self.gdb_port)])
+                          ['-ex', 'target remote {}:{}'.format(self.gdb_host, self.gdb_port)])
             if command == 'debug':
                 client_cmd += ['-ex', 'monitor halt',
                                '-ex', 'monitor reset',
                                '-ex', 'load']
                 if self.reset_after_load:
                     client_cmd += ['-ex', 'monitor reset']
-
-            self.print_gdbserver_message()
-            self.run_server_and_client(server_cmd, client_cmd)
+            if not self.gdb_host:
+                self.print_gdbserver_message()
+                self.run_server_and_client(server_cmd, client_cmd)
+            else:
+                previous = signal.signal(signal.SIGINT, signal.SIG_IGN)
+                self.check_call(client_cmd)
+                signal.signal(signal.SIGINT, previous)
```

Comment:
Why isn't this using `popen_ignore_int`?

Reply:
I did not know that existed, will change.

Reply:
Actually using `popen_ignore_int` wasn't what I needed, as this would have spawned a child process. I wanted to run a command while ignore `SIGINT`, like it was done in `run_server_and_client`.
I've split this in a `run_client` which hopefully shows the intent.